cmd/promtool/main.go

```diff
@@ -86,6 +89,9 @@ func main() {
 		"The rule files to check.",
 	).Required().ExistingFiles()
 
+	checkExpressionsCmd := checkCmd.Command("expressions", checkExpressionsUsage)
```

Comment:
since we decided that this subcommand will only accept 1 expression at a time, can we replace all places it says `expressions` with `expression`?

s/expressions/expression/g